dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -13,7 +13,7 @@
     "owner": "airflow",
     "description": "Create/update 'Index' issues in the DTS portal from Github.",
     "depends_on_past": False,
-    "start_date": datetime(2015, 12, 1),
+    "start_date": pendulum.datetime(2015, 12, 1, tz="America/Chicago"),
```

Comment:
I keep wondering a time will come when a DAG wrapper that would set config that we would want to apply to every single DAG ever would make sense, but I think it would be overkill tbh.